cssdb.settings.json

```diff
@@ -804,24 +804,9 @@
     },
     "example": "span:first-child {\n  margin-inline-start: 10px;\n}",
     "mdn_path": [
-      "css.properties['inline-size']",
-      "css.properties['inline-size']['max-content']",
-      "css.properties['inline-size']['min-content']",
-      "css.properties['min-inline-size']",
-      "css.properties['min-inline-size']['max-content']",
-      "css.properties['min-inline-size']['min-content']",
-      "css.properties['max-inline-size']",
-      "css.properties['max-inline-size']['max-content']",
-      "css.properties['max-inline-size']['min-content']",
       "css.properties['block-size']",
-      "css.properties['block-size']['max-content']",
-      "css.properties['block-size']['min-content']",
-      "css.properties['min-block-size']",
-      "css.properties['min-block-size']['max-content']",
-      "css.properties['min-block-size']['min-content']",
-      "css.properties['max-block-size']",
-      "css.properties['max-block-size']['max-content']",
-      "css.properties['max-block-size']['min-content']",
+      "css.properties['border-block']",
```

Comment:
I've sorted these properties and removed `max-content` and `min-content` variants.
We don't do anything for these features (correct me if I am wrong).

Omitting them better fits our policy to describe the browser support required for the polyfill/fallback.

Reply:
Thanks for cleaning and sorting these!